src/Interpreters/InterpreterCreateQuery.cpp

```diff
@@ -869,6 +869,26 @@ InterpreterCreateQuery::TableProperties InterpreterCreateQuery::getTableProperti
         {
             properties.indices = as_storage_metadata->getSecondaryIndices();
             properties.projections = as_storage_metadata->getProjections().clone();
+
+            /// CREATE TABLE AS should copy PRIMARY KEY, ORDER BY, and similar clauses.
+            /// Note: only supports the source table engine is using the new syntax.
```

Comment:
The grammar of the sentence in l. 874 is somehow broken.

Reply:
Is it possible to modify the comment description to `/// Note: Only the source table definition with the new syntax is supported.`? If it's okay, I'd like to update this comment in subsequent PRs.

Reply:
Yes, let's fix in a subsequent PR.